test_local_changes.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+
+import Bundle_ASH_script
+
+# If needed, replace this with the path to your mafia's base config folder
+MAFIA_DIRECTORY = f"/Users/{os.getlogin()}/Library/Application Support/Kolmafia"
```

Comment:
Not sure if Mafia has an easy default to add here for Windows, but happy to add it (and possibly OS-detection too?) if so.

Reply:
it does not unfortunately. i think maybe we'll want to make the test_local_changes.py file have an argument on run that references a directory. pretty sure this definitely isn't where mine is though i'll check later. i am on windows so i can be a guinea pig here :-)